### PR TITLE
use logging slice from http 0.8

### PR DESCRIPTION
**Pull request by brastak**
For #4

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#11](https://github.com/artipie/npm-proxy-adapter/issues/11) is assigned to @hdouss[/z](https://www.0crat.com/u/hdouss), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/a24d54b3-ba66-4dd3-b75f-12b0cf2a7908); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/npm/proxy/http/RsNotFound.java [hunk 1]
```diff
@@ -41,8 +42,10 @@
     public CompletionStage<Void> send(final Connection connection) {
         return connection.accept(
             RsStatus.NOT_FOUND,
-            new ListOf<Map.Entry<String, String>>(
-                new MapEntry<>("Content-Type", "application/json")
+            new Headers.From(
```
Comment (HDouss) [inline src/main/java/com/artipie/npm/proxy/http/RsNotFound.java:45]: @brastak is this related to LoggingSlice? What does it add?

Comment (brastak) [reply src/main/java/com/artipie/npm/proxy/http/RsNotFound.java:45]: @HDouss it's related with upgrade to http 0.8. Signature was changed

**Review by HDouss — APPROVED** [review]
@brastak Thanks.

**Comment by HDouss** [comment]
@g4s8 Please review or merge.

**Comment by HDouss** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/npm-proxy-adapter/issues/11#issuecomment-618593506))

@HDouss This is what I know about this job in CT2E6TK9B, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#11](https://github.com/artipie/npm-proxy-adapter/issues/11) is [in scope](http://www.zerocracy.com/policy.html#14) for 2days
  * The role is `REV`
  * The job is assigned to @hdouss[/z](https://www.0crat.com/u/hdouss) for [2days](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @hdouss[/z](https://www.0crat.com/u/hdouss) will get 25% of their hourly rate for this job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @brastak[/z](https://www.0crat.com/u/brastak): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/CT2E6TK9B?q=%7Bjob%3A%27gh%3Aartipie%2Fnpm-proxy-adapter%2311%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by HDouss** [comment]
@g4s8 how to trigger rewards here?

**Comment by g4s8** [comment]
@0crat out

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @hdouss[/z](https://www.0crat.com/u/hdouss), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@HDouss please make sure you will find at least 3 issues during next CR, thanks!

**Comment by HDouss** [comment]
@sereshqua Sure, this one has little relevant part, so I only found 1 comment to do.

**Comment by sereshqua** [comment]
@0crat quality acceptable
